### PR TITLE
Updated windows installation docs

### PR DESCRIPTION
**Pull request by dominikl**
Updates the Windows installation documentation following @manics installation scripts.

Deployed on https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/server-installation.html

A few notes:
- I don't know if/what kind of update the other windows pages need, e. g. https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/service-tools.html
- @aleksandra-tarkowska : Is the remark about unsetting `omero.web.session_engine` and `omero.web.caches` still valid for Django 1.8? - clarified, still needed

**Depends on:** https://github.com/openmicroscopy/openmicroscopy/pull/4368

### CODE REVIEW

File: omero/sysadmins/windows/server-installation.txt [hunk 1]
```diff
@@ -124,6 +127,10 @@ The following are optional depending on what services you require:
       - Functionality
       - Downloads
 
+    * - Django (1.8.5)
```
Comment (atarkowska) [inline omero/sysadmins/windows/server-installation.txt]: Could we change to 1.8.x because it is LTS

Comment (atarkowska) [inline omero/sysadmins/windows/server-installation.txt]: Django is not additional library really. I think is enough to mention pip install on IIS page.

File: omero/sysadmins/windows/server-installation.txt [hunk 2]
```diff
@@ -124,6 +127,10 @@ The following are optional depending on what services you require:
...
+      - OMERO.web
+      - `Django website <https://www.djangoproject.com/>`_
```
Comment (atarkowska) [inline omero/sysadmins/windows/server-installation.txt]: more appropriate link `Django 1.8 installation on Windows https://docs.djangoproject.com/en/1.8/howto/windows/`_

**Comment by atarkowska** [comment]
@dominikl thanks for the update, could we also add under screenshot _IIS 7 Application Pool Advanced Settings_ note block **if deploying on 64-bit Enable 32 bit applications setting set back to False**?

**Comment by dominikl** [comment]
I'm currently using windows server 2012 for testing (as windows 8 desktop doesn't work that well), which has a little bit different GUI, so the screenshot would look a bit different, if that's not a problem I can add a screenshot. But shouldn't the **Enable 32 bit** part be set to **True**!?

**Comment by dominikl** [comment]
And I'm not sure what to do with this "additional Libraries" section. Are these all libraries for OMERO.web? Then I could move the whole section to iis-install doc. If it's just meant for the OMERO.server Django would actually be an optional library. This section is really more confusing than helpful.

**Comment by atarkowska** [comment]
> But shouldn't the Enable 32 bit part be set to True!?

I think If you install 64-bit python, it is no longer 32-bit application that's why should be false. On WindowsServer2012 with Python 2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)] web is not working when processes in WOW64 are in 32bit mode. The same was said https://www.openmicroscopy.org/community/viewtopic.php?f=5&t=7937&hilit=Web+windows#p16444

**Comment by atarkowska** [comment]
> Is the remark about unsetting `omero.web.session_engine` and `omero.web.caches` still valid for Django 1.8?

As we don't know which version people update from, I would leave it.

**Comment by manics** [comment]
I thought we dropped support for 32 bit OMERO.server?

**Comment by atarkowska** [comment]
> https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/server-installation.html  I just ran through the doc again, now at the last step, starting omero.web in development mode. if you follow the instruction you get stuck there with error "django not installed"
> so i'll just add the instruction `pip install -r share/web/requirements-py27-nginx.txt` there was well.

I renamed requirements file to be more usable across the doc https://github.com/openmicroscopy/openmicroscopy/pull/4368. Could you add `pip install -r share/web/requirements-py27-win.txt`

File: omero/sysadmins/windows/install-web/install-iis.txt [hunk 3]
```diff
@@ -9,17 +9,17 @@ IIS configuration (Windows)
...
        see :ref:`python-requirements` on the
        :doc:`/sysadmins/version-requirements` page.
 
-Install `Django 1.6`_ using package requirements file:
+Install `Django 1.8`_ using package requirements file:
 
 ::
 
-   $ pip install -r share/web/requirements-py26-iis.txt
+   $ pip install -r share/web/requirements-py27-iis.txt
```
Comment (atarkowska) [inline omero/sysadmins/windows/install-web/install-iis.txt]: `pip install -r share/web/requirements-py27-win.txt`

**Comment by dominikl** [comment]
Just checked, one really doesn't need to enable the 32bit feature. And it doesn't have to be set to "false", it's "false" as default.

**Comment by dominikl** [comment]
Ah sorry, no I understand what you meant @aleksandra-tarkowska , the screenshot is still showing the enabled 32bit option. I'll add a new screenshot.

**Comment by jburel** [comment]
need to indicate that pillow version is 2.9.0
see https://github.com/ome/figure/pull/136

**Comment by joshmoore** [comment]
A passing comment having not looked at this deeply: _very nice_.

File: omero/sysadmins/windows/install-web/install-iis.txt [hunk 4]
```diff
@@ -71,6 +71,12 @@ working with your IIS deployment.
    :align: center
    :alt: IIS 7 HTTP response header
 
+-  Make sure the OMERO's log directory has full read/write permissions for all users
```
Comment (mtbc) [inline omero/sysadmins/windows/install-web/install-iis.txt]: "Make sure _that_ OMERO's ... " perhaps?

**Comment by dominikl** [comment]
**Review** last commit only, deployed on https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/install-web/install-iis.html

**Comment by hflynn** [comment]
Merge build running now, last commit will be built in about 10 mins

**Comment by atarkowska** [comment]
@dominikl thanks looks good

**Comment by ghost** [comment]
Looks pretty good to me.

**Comment by hflynn** [comment]
Waiting on code PR being merged before I merge this.

**Comment by atarkowska** [comment]
@hflynn we also need to merge https://github.com/openmicroscopy/openmicroscopy/pull/4296. I wasn't able to fix Windows build as I have no rights on burbot and don't know anything about omego.

**Comment by hflynn** [comment]
Both linked PRs merged so merging

**Pull request merged**